<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сохранение заявки</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>501</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Bahnschrift&quot;;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Партнеры</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="partners_tableView">
      <property name="minimumSize">
       <size>
        <width>411</width>
        <height>201</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(187, 220, 250);
font: 9pt &quot;Bahnschrift&quot;;
color: rgb(0, 0, 0);</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="exit_pushButton">
        <property name="minimumSize">
         <size>
          <width>121</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(12, 72, 130);
font: 9pt &quot;Bahnschrift&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="save_pushButton">
        <property name="minimumSize">
         <size>
          <width>121</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(12, 72, 130);
font: 9pt &quot;Bahnschrift&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
